src/main/js/store/actions/system.js

```diff
@@ -120,9 +120,10 @@ export function getLatestMessages() {
         fetch(LATEST_MESSAGES_URL)
             .then((response) => {
                 if (response.ok) {
-                    response.json().then((body) => {
-                        dispatch(latestSystemMessagesFetched(body));
-                    });
+                    response.json()
```

Comment:
Most of the changes in here are just formatting. You might want to double check your JS save actions.

Reply:
The SaveActions in Intellij are intended for Java and often mess up our JS files because of it. I usually disable the save actions plugin when working in JS land.